subsys/bluetooth/controller/ll_sw/pdu.h

```diff
@@ -208,6 +208,33 @@
 
 #define PKT_BIS_US(octets, mic, phy) PDU_MAX_US((octets), (mic), (phy))
 
+/* EGON TODO: verify if the following lines are correct */
```

Comment:
What is EGON? 

Have these been verified by now?

Reply:
See issue #15256
One of the remaining action items is to go through all TODOs and evaluate what is to be done with these